django/conf/global_settings.py

```diff
@@ -628,6 +628,15 @@ def gettext_noop(s):
 # The default file storage backend used during the build process
 STATICFILES_STORAGE = "django.contrib.staticfiles.storage.StaticFilesStorage"
 
+# File storage info. If left empty, will default to DEFAULT_FILE_STORAGE and
+# STATICFILES_STORAGE.
+DEFAULT_STORAGE_ALIAS = "default"
+STATICFILES_STORAGE_ALIAS = "staticfiles"
+STORAGES = {
+    DEFAULT_STORAGE_ALIAS: {"BACKEND": DEFAULT_FILE_STORAGE},
+    STATICFILES_STORAGE_ALIAS: {"BACKEND": STATICFILES_STORAGE},
```

Comment:
This should be prepared for removing `DEFAULT_FILE_STORAGE` and `STATICFILES_STORAGE`
```suggestion
    "default": {"BACKEND": "django.core.files.storage.FileSystemStorage"},
    "staticfiles": {"BACKEND": "django.contrib.staticfiles.storage.StaticFilesStorage"},
```
`Settings` will keep them in sync.

Reply:
Moved `STORAGES` setting defaults to `Settings.__init__` in `django/conf/__init__.py`.